packages/connector-local/src/connector.ts

```diff
@@ -242,6 +245,17 @@ export default class LocalConnector implements IConnector {
         });
     }
 
+    private watchForWindow() {
+        this._window = null;
+
+        const setWindow = (event: HTMLParse) => {
+            this._window = event.window;
```

Comment:
This also needs to emit a `can-evaluate::script` event here or `hint-axe` and friends won't work. I'll update the PR for this in a bit (need to grab a bite to eat).